content/php-client/ce-resources/media-files.md

```diff
@@ -78,3 +78,16 @@ $client->getProductMediaFileApi()->create('/tmp/ziggy.jpg', [
 ]);
 ```
 
+To create a new media file and associate it to a product model, you have to set the property `type` and the code of the product model.
```

Comment:
I would separate the content into two subsections here.
 `#### Association to a product` and 
 `#### Association to a product model` 
With the following sentence to introduce these sections: 'When you create a media file, you can directly associate to either a product or a product model'